openspec/changes/archive/2026-04-17-slash-command-transport-parity/exploration.md

```
@@ -0,0 +1,52 @@
+## Exploration: issue #541 slash command transport parity
+
+### Current State
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)